### PR TITLE
MTM-56417 cleanup managing applications doc 10 18

### PR DESCRIPTION
**Pull request by eniosultan**

### CODE REVIEW

File: content/users-guide/administration-bundle/managing-applications.md [hunk 1]
```diff
@@ -300,17 +300,41 @@ In the **Properties** tab, each application will show the following information,
 </table>
 
 <a name="extensions"></a>
-### Extensions
+### Extensions {#packages}
 
-Click **Extensions** in the **Ecosystem** menu in the navigator to display all extensions in your account.
+Extensions or packages are combinations of plugins and blueprints which can be packed together into a single file and then be deployed to the platform. Thus, they offer better shareability and reusability of UI features across different applications and allow to add UI features to applications without coding knowledge.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-applications.md:305]: Here you say "Extensions and packages" while below you talk about "Extension packages". What is the correct terminology? Please explain how these terms are used here.

File: content/users-guide/administration-bundle/managing-applications.md [hunk 2]
```diff
@@ -300,17 +300,41 @@ In the **Properties** tab, each application will show the following information,
 </table>
 
 <a name="extensions"></a>
-### Extensions
+### Extensions {#packages}
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-applications.md:303]: Why do you use an ID which is different from the actual heading? Please change this to #extensions.

**Comment by BeateRixen** [comment]
@eniosultan Please also check the 10.17 documentation. I know that it is different (not having extensions, I believe) but it has two sections with the heading "Packages"

File: content/users-guide/administration-bundle/managing-applications.md (hunk 1 above)
Comment (BeateRixen) [reply content/users-guide/administration-bundle/managing-applications.md:305]: But if you say "Extensions or packages" then the user assumes that these are two different things. Why do you make such a terminology mess :-). I suggest to use Extension packages then also in the first sentence.

File: content/users-guide/administration-bundle/managing-applications.md [hunk 3]
```diff
@@ -300,17 +300,41 @@ In the **Properties** tab, each application will show the following information,
 </table>
 
 <a name="extensions"></a>
-### Extensions
+### Extensions {#extensions}
 
-Click **Extensions** in the **Ecosystem** menu in the navigator to display all extensions in your account.
+Extensions or packages are combinations of plugins and blueprints which can be packed together into a single file and then be deployed to the platform. Thus, they offer better shareability and reusability of UI features across different applications and allow to add UI features to applications without coding knowledge.
```
Comment (eniosultan) [inline content/users-guide/administration-bundle/managing-applications.md:305]:
```suggestion
Extensions packages are combinations of plugins and blueprints which can be packed together into a single file and then be deployed to the platform. Thus, they offer better shareability and reusability of UI features across different applications and allow to add UI features to applications without coding knowledge.
```

File: content/users-guide/administration-bundle/managing-applications.md (hunk 1 above)
Comment (eniosultan) [reply content/users-guide/administration-bundle/managing-applications.md]: Fixed

File: content/users-guide/administration-bundle/managing-applications.md [hunk 4]
```diff
@@ -300,17 +300,41 @@ In the **Properties** tab, each application will show the following information,
 </table>
 
 <a name="extensions"></a>
-### Extensions
+### Extensions {#extensions}
 
-Click **Extensions** in the **Ecosystem** menu in the navigator to display all extensions in your account.
+Extensions packages are combinations of plugins and blueprints which can be packed together into a single file and then be deployed to the platform. Thus, they offer better shareability and reusability of UI features across different applications and allow to add UI features to applications without coding knowledge.
```
Comment (BeateRixen) [inline content/users-guide/administration-bundle/managing-applications.md:305]:
```suggestion
Extension packages are combinations of plugins and blueprints which can be packed together into a single file and then be deployed to the platform. Thus, they offer better shareability and reusability of UI features across different applications and allow to add UI features to applications without coding knowledge.
```

**Review by BeateRixen — APPROVED** [review]

**Review by jdreSAG — APPROVED** [review]

**Pull request merged**